my-portfolio/src/components/pdfViewer.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+const PdfViewer = () => {
+  return (
+    <div className="pdfViewer">
+      <iframe src="\cv.pdf" width="60%" height="1200px" />
```

Comment:
`"\cv.pdf"` I concerned about `\` why did you use it? Windows?

Reply:
it's because a copied the path ... i fixed it